generated/nixnetsocket/nixnetsocket.proto

```diff
@@ -148,7 +148,7 @@ message GatewayAddress {
 
 message VirtualInterface {
   string xnet_interface_name = 1;
-  string vlane_name = 2;
+  string vlan_name = 2;
```

Comment:
fyi: I already made this change.

Reply:
Mose had mentioned you've made some updates so I'll be sure to handle any merge conflicts / remove this from mine